### PR TITLE
Issue517 and issue518

### PR DESCRIPTION
**Pull request by robstryker**

### CODE REVIEW

File: framework/bundles/org.jboss.tools.rsp.runtime.core/src/main/java/org/jboss/tools/rsp/runtime/core/model/installer/internal/DownloadRuntimeOperationUtilFactory.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package org.jboss.tools.rsp.runtime.core.model.installer.internal;
```
Comment (odockal) [inline framework/bundles/org.jboss.tools.rsp.runtime.core/src/main/java/org/jboss/tools/rsp/runtime/core/model/installer/internal/DownloadRuntimeOperationUtilFactory.java]: Copyright please.

**Comment by odockal** [comment]
I tried to download both CDK and CRC, CDK was downloaded successfully - with progress bar working.

But CRC did not show progress and eventually timed out. On the second trial, it is downloading and showing progress... strange, will investigate more.

**Comment by odockal** [comment]
@robstryker Aaaand.. it is working now.

File: framework/bundles/org.jboss.tools.rsp.runtime.core/src/main/java/org/jboss/tools/rsp/runtime/core/model/installer/internal/DownloadRuntimeOperationUtilFactory.java (hunk 1 above)
Comment (robstryker) [reply framework/bundles/org.jboss.tools.rsp.runtime.core/src/main/java/org/jboss/tools/rsp/runtime/core/model/installer/internal/DownloadRuntimeOperationUtilFactory.java]: I added copyright and removed some unused imports. I have no intention to rewrite some of the methods to use try-with-resources nor do i intend to change method signatures due to an unused variable at this time.

**Pull request merged**